src/mds/cephfs_features.h

```diff
@@ -58,6 +59,7 @@ namespace ceph {
   CEPHFS_FEATURE_DELEG_INO,             \
   CEPHFS_FEATURE_OCTOPUS,               \
   CEPHFS_FEATURE_METRIC_COLLECT,        \
+  CEPHFS_FEATURE_FSCRYPTED,             \
```

Comment:
I wonder if we ought to name this something more generic. While we're adding this for fscrypt, it's potentially useful for other things too. Maybe CEPHFS_FEATURE_ALT_DNAME ?

Reply:
Yeah, sounds a good idea, so all the others, like the fake attr key string `fscrypt.alternate_name`, all should be change to more generic.